core/store/src/trie/shard_tries.rs

```diff
@@ -155,19 +155,16 @@ impl ShardTries {
         store_update: &mut StoreUpdate,
     ) {
         store_update.tries = Some(tries);
-        for TrieRefcountChange { trie_node_or_value_hash, trie_node_or_value, rc } in
-            deletions.iter()
-        {
+        for TrieRefcountChange { trie_node_or_value_hash, rc, .. } in deletions.iter() {
+            let rc = match std::num::NonZeroU32::new(*rc) {
+                None => continue,
+                Some(rc) => rc,
+            };
```

Comment:
We might want to potentilaly push non-zero checks to 

https://github.com/matklad/nearcore/blob/ef8884ca0efecb36487640818db8db03831b72c2/core/store/src/trie/mod.rs#L708-L721

?